app/views/assessments/index.html.erb

```diff
@@ -54,13 +54,16 @@
        <h3 style="border-bottom: 4px solid #dddddd"></h3>
           <p style="font-size: 18px">Section: <b><%= "#{@cud.section}" %></b></p>
           <p style="font-size: 18px">Number of grace days remaining: <b><%="#{@cud.global_grace_days_left}"%></b></p></br>
+
           <% if @course.website.nil? || @course.website.eql?("") %>
-          <span class="btn btn-large disabled"><%= "<i class='material-icons left'>web</i> Course Website".html_safe %></span>
+            <span class="btn btn-large disabled"><%= "<i class='material-icons left'>web</i> Course Website".html_safe %></span>
           <% else %>
-          <%= link_to "<i class='material-icons left'>web</i> Course Website".html_safe, @course.website, target: "_blank", title: "Go to course website", :class => "btn btn-large red darken-3" %>
+            <%= link_to "<i class='material-icons left'>web</i> Course Website".html_safe, @course.website, target: "_blank", title: "Go to course website", :class => "btn btn-large red darken-3" %>
           <% end %>
-          <% if @cud.instructor? || !(@cud.course_assistant? && !@cud.section.blank?) %>
```

Comment:
In the previous logic, we have set it up so CA's that have an assigned section cannot view the gradebook. I believe this is because some instructors prefer to give certain CA's limited visibility (i.e only have access to grading their section and not viewing all of the students grades). So, I think this logic is still needed for the gradebook.